kong/db/migrations/operations/331_to_332.lua

```diff
@@ -0,0 +1,65 @@
+-- Helper module for 331_to_332 migration operations.
+--
+-- Operations are versioned and specific to a migration so they remain
+-- fixed in time and are not modified for use in future migrations.
+--
+-- If you want to reuse these operations in a future migration,
+-- copy the functions over to a new versioned module.
+
+
+local function render(template, keys)
+  return (template:gsub("$%(([A-Z_]+)%)", keys))
+end
+
+
+--------------------------------------------------------------------------------
+-- Postgres operations for Workspace migration
+--------------------------------------------------------------------------------
+
+
+local postgres = {
+
+  up = {},
+
+  teardown = {
+
+    ------------------------------------------------------------------------------
+    -- General function to fixup a plugin configuration
+    fixup_plugin_config = function(_, connector, plugin_name, fixup_fn)
+      local pgmoon_json = require("pgmoon.json")
+      for plugin, err in connector:iterate("SELECT id, name, config FROM plugins") do
```

Comment:
Would it be beneficial to have a WHERE clause here? E.g. `WHERE name = 'opentelementry'`

Reply:
done, thanks